app/javascript/src/Policies/components/PolicyChain.jsx

```diff
@@ -28,26 +28,24 @@ const SortableItem = SortableElement(({value, editPolicy, index}) => {
   const edit = () => editPolicy(value, index)
   return (
     <li className={ value.enabled ? 'Policy' : 'Policy Policy--disabled' }>
-      <PolicyTile policy={value} onClick={edit} />
+      <PolicyTile policy={value} onClick={edit} title="Edit this Policy" />
       <DragHandle/>
     </li>
   )
 })
 
-const SortableList = SortableContainer(({ items, editPolicy }) => {
-  return (
-    <ul className="list-group">
-      {items.map((policy, index) => (
-        <SortableItem
-          key={`item-${index}`}
-          index={index}
-          value={policy}
-          editPolicy={editPolicy}
-        />
-      ))}
-    </ul>
-  )
-})
+const SortableList = SortableContainer(({ items, editPolicy }) => (
```

Comment:
These kind of changes are always quite annoying... nothing really changes here, just returning implicitly... would be nice if GH can avoid marking as all lines has changed...

Reply:
Yes it would..